### PR TITLE
Added Name Krishna Bagadia to README.md

### PR DESCRIPTION
**Pull request by krishna95**
Added my Name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
-Harsh Gupta <gupta.harsh96@gmail.com>
+Harsh Gupta <gupta.harsh96@gmail.com>
```
Comment (hargup) [inline README.md]: There is a trailing white space at the end of the line, remove it.

**Comment by hargup** [comment]
Better commit message please, reading the logs should tell me which name was added.

File: README.md (hunk 1 above)
Comment (abinashmeher999) [inline README.md]: How do you know if there's a trailing white space?

**Comment by hargup** [comment]
@krishna95 see https://github.com/AGV-IIT-KGP/git-sandbox/pull/8#issuecomment-75957862. Squash the two commits and this PR will be good to go.

**Comment by krishna95** [comment]
@hargup Please Review

File: README.md [hunk 2]
```diff
@@ -7,4 +7,6 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
+Krishna Bagadia <krishna.bagadia2@gmail.com><br/>
```
Comment (abinashmeher999) [inline README.md]: @krishna95 Since when K came before H?

**Comment by abinashmeher999** [comment]
@krishna95 Still appearing on the same line. Fix it.

**Comment by hargup** [comment]
There are also merge conficts, resolve them. And you might want to do it before the @abinashmeher999 's [PR](https://github.com/AGV-IIT-KGP/git-sandbox/pull/7) gets merged :smile: .

**Comment by krishna95** [comment]
@hargup  Please Review

**Comment by hargup** [comment]
Squash your commits into one.

**Comment by krishna95** [comment]
@hargup  done (Y)

**Comment by hargup** [comment]
Thanks, it is in.

**Pull request merged**